tests/testthat/helpers-external.R

```diff
@@ -0,0 +1,116 @@
+#' Create a temporary directory with external results
+#'
+#' @param pattern,clean,.local_envir Arguments passed to
+#'   `withr::local_tempdir()`.
+local_create_external_results <- function(pattern = "mpn-scorecard-tests-",
+                                          .local_envir = parent.frame(),
+                                          clean = TRUE) {
+  tdir <- withr::local_tempdir(
+    pattern = pattern, .local_envir = .local_envir, clean = clean
+  )
```

Comment:
Could export `local_create_external_results()` (perhaps with different name), and reference in `external_scores` documentation to see an example. Just thinking out loud

Reply:
Thanks for the suggestion.  My preference at this point would be to avoid adding something to the NAMESPACE that has unclear value.  If that usefulness becomes clear later, it's easy to move/rename/export.

> Though exporting `local_create_external_results` might be overkill, it would be nice to provide more support for setting this up, especially given the required file naming conventions.

I don't think these are files that anyone would want to set up by hand (with the exception of the traceability yaml).  You'd always have tooling to generate these automatically.